app/assets/stylesheets/components/_alert-icon.scss

```
@@ -22,3 +22,9 @@
     background-image: url('status/personal-key.svg');
   }
 }
+.alert-icon-centered-top {
```

This looks like it could be a [modifier](https://en.bem.info/methodology/key-concepts/#modifier) in our [convention of using BEM methodology](https://github.com/18F/identity-idp/blob/main/docs/frontend.md#css--html) (docs could be clearer, admittedly):

```suggestion
.alert-icon--centered-top {
```

---

Also I just noticed the login design system link is broken in the Frontend README (it shows https://design.login.gov/components/)

---

> Also I just noticed the login design system link is broken in the Frontend README (it shows https://design.login.gov/components/)

Good catch! (Though personally that sounds more like an issue with the Design System docs not having a good redirect for that path, since I think it _should_ be a valid URL)